src/h2/events.py

```diff
@@ -589,11 +589,9 @@ def __init__(self):
         self.field_value = None
 
     def __repr__(self):
-        return (
-            "<AlternativeServiceAvailable origin:%s, field_value:%s>" % (
-                self.origin.decode('utf-8', 'ignore'),
-                self.field_value.decode('utf-8', 'ignore'),
-            )
+        return "<AlternativeServiceAvailable origin:%s, field_value:%s>" % (
+            self.origin.decode("utf-8", "replace"),
+            self.field_value.decode("utf-8", "replace"),
```

Comment:
why the change in decoding from ignore to replace? If this is an unrelated change, please revert it and let's discuss in a new issue or PR.

Reply:
Unrelated, reverting.

Reply:
Please revert.

Reply:
Whoops, sorry. Should be reverted now